src/main/java/frc/robot/Controls.java

```diff
@@ -340,4 +340,23 @@ private void configureElevatorLEDBindings() {
           s.elevatorLEDSubsystem.colorSet(50, 0, 0).withName("LED red").ignoringDisable(false));
     }
   }
+
+  private void configureAlignBindings() {
+    if (s.drivebaseSubsystem == null) {
+      return;
+    }
+    driverController.leftBumper().onTrue(AutoAlign.autoAlign(s.drivebaseSubsystem));
```

Comment:
Needs to go in after #43 that moves the existing `leftBumper` control (see line 165 in this version of the file) to a different button.